### PR TITLE
PELE-451 Support for conformation perturbtation

### PR DESCRIPTION
**Pull request by frumpowy**
Compatible with peleffy PR.

### CODE REVIEW

**Review by martimunicoy — APPROVED** [review]

File: docs/build_docs/source/flags/all_packages/index.rst [hunk 1]
```diff
@@ -289,6 +289,20 @@ Alternatively, as before, you can provide your own template and/or rotamer files
...
+Ligand conformations
+----------------------
+
+PELE provides the possibility to narrow down the range of available ligand conformations to increase the efficiency of
+sampling. It will automatically generate a library of conformations when supplied with a directory of ligand clusters
+originating from conformational search or `Bioactive Conformational Ensemble server <https://mmb.irbbarcelona.org/BCE/>`_.
+
+- **ligand_conformations**: Path to the directory contraining ligand clusters in PDB format.
+
+- **conformation_freq**: Frequency of conformation perturbation. Default = 0.
```
Comment (martimunicoy) [inline docs/build_docs/source/flags/all_packages/index.rst:301]: Should be 4

File: pele_platform/Utilities/Parameters/SimulationParams/simulation_params.py [hunk 2]
```diff
@@ -111,6 +113,11 @@ def main_pele_params(self, args):
             if args.water_freq is not None
             else self.simulation_params.get("water_freq", 1)
         )
+        self.conformation_freq = (
+            args.conformation_freq
+            if args.conformation_freq is not None
+            else self.simulation_params.get("conformation_freq", 0)
```
Comment (martimunicoy) [inline pele_platform/Utilities/Parameters/SimulationParams/simulation_params.py:119]: the same here, it should be 4

**Review by martimunicoy — CHANGES_REQUESTED** [review]

File: pele_platform/Utilities/Parameters/SimulationParams/simulation_params.py (hunk 2 above)
Comment (frumpowy) [inline pele_platform/Utilities/Parameters/SimulationParams/simulation_params.py:119]:
```suggestion
            else self.simulation_params.get("conformation_freq", 4)
```

File: docs/build_docs/source/flags/all_packages/index.rst (hunk 1 above)
Comment (frumpowy) [inline docs/build_docs/source/flags/all_packages/index.rst:301]:
```suggestion
- **conformation_freq**: Frequency of conformation perturbation. Default = 4.
```

**Review by martimunicoy — APPROVED** [review]

**Pull request merged**